manifests/07_deployment.yaml

```diff
@@ -23,10 +23,14 @@ spec:
       - name: operator
         image: quay.io/openshift/origin-cluster-authentication-operator:v4.0
         imagePullPolicy: IfNotPresent
-        command: ["authentication-operator", "operator"]
+        command: ["/bin/bash", "ec"]
         args:
-        - "--config=/var/run/configmaps/config/operator-config.yaml"
-        - "-v=2"
+        - |
+          if [ -s /var/run/configmaps/trusted-ca-bundle/tls-ca-bundle.pem ]; then
```

Comment:
I wish the operator does this during startup and we don't have to hack ways in bash.

Reply:
it cannot. Then it is too late. And this does not belong into a binary. It is system plumbing. It is just right here.

Reply:
if `tls-ca-bundle.pem` is empty, will that brick the operator?

Reply:
it is non-existent if not part of the configmap.

Reply:
The network operator validates it.

Reply:
@stlaz noticed that we must not copy empty files if I have understood him correctly.